contrib/mpi-proxy-split/mpi-wrappers/p2p-deterministic.txt

```diff
@@ -164,9 +167,9 @@ ssize_t writeall(int fd, void *buf, size_t count) {
   }
 
 // Before call to MPI_Irecv() in lower half during replay
-// NOT USED:  MPI_Recv wrapper calls MPI_Irecv
+// FIXME:  Need to add MPI_Recv_init when MANA supports it
 #define REPLAY_PRE_Irecv(count,datatype,source,tag,comm)
-  if (getenv("MANA_P2P_REPLAY") && !MPI_Iprobe_recurse) {
+  if (getenv("MANA_P2P_REPLAY")) {
     struct p2p_log_msg p2p_msg;
     while (!iprobe_next_msg(NULL)) {
```

Comment:
Currently `iprobe_next_msg` does not work when NULL is passed in. We may need to fix the function needs to open log file and read the first message.